packages/react-client/src/hooks/devices/useMicrophone.ts

```diff
@@ -0,0 +1,19 @@
+import type { AudioDevice } from "../../types/public";
+import { useFishjamContext } from "../useFishjamContext";
+import { useDeviceApi } from "./useDeviceApi";
+
+/**
+ *
+ * @category Devices
+ */
+export function useMicrophone(): AudioDevice {
+  const { audioTrackManager, audioDeviceManagerRef } = useFishjamContext();
+
+  const api = useDeviceApi({ trackManager: audioTrackManager, deviceManager: audioDeviceManagerRef.current });
+  const isAudioPlaying = audioTrackManager.currentTrack?.vadStatus === "speech";
```

Comment:
Curious: will work in a way, developer would expect - so `isAudioPlaying` will be updated in component always when `vadStatus` will change?

Reply:
Well, good point. After brief consideration - it shouldn't. Changes can be rapid and it doesn't make much sense to re-render on **own** VAD status.
To fix this issue I created [FCE-720](https://linear.app/swmansion/issue/FCE-720/create-usevad-hook-to-allow-for-optional-listening-to-vad-updates) ticket and I queue it as my next task.